### PR TITLE
[AIRFLOW-407] Add different colors for some sensors

### PR DESCRIPTION
**Pull request by ldct**
Dear Airflow Maintainers,

Please accept this PR that addresses the following issues:
- [https://issues.apache.org/jira/browse/AIRFLOW-407](https://issues.apache.org/jira/browse/AIRFLOW-407)

These colors were added (leftmost is the default sensor color):

image
#### Testing
- [x] Create a DAG with all these operators and take a screenshot

### CODE REVIEW

File: airflow/operators/sensors.py [hunk 1]
```diff
@@ -256,6 +259,7 @@ class NamedHivePartitionSensor(BaseSensorOperator):
     """
 
     template_fields = ('partition_names', )
+    ui_color = '#2b2d42'
```
Comment (aoen) [inline airflow/operators/sensors.py]: Factor this out since it's reused

**Comment by aoen** [comment]
Fix and LGTM

File: airflow/operators/sensors.py (hunk 1 above)
Comment (ldct) [inline airflow/operators/sensors.py]: oh no, it should not be reused...wrong colour

**Comment by ldct** [comment]
image

**Pull request closed without merging**